hack/development-environment/kind-local-up.sh

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
-set -e
```

Comment:
Are there specific lines you want to allow to fail?  Might be more targeted to add `|| true` to those.

Reply:
I thought the same thing.. but  not really, that was just for debugging. 
Also don't want to complicate the readability of the script.
I think we can make a more defensive bash script in the future though, possibly w/ "real" command line options as well, if you want i can hack on that for the future? 

If you insist, though, ill re-add the `-e`.

Reply:
Not sure I get the point here.  If you expect no errors in the script, surely it's good practice to keep `-e`?  If a line failed unexpectedly, what could be the benefit of the script continuing after that point?

Reply:
classic tradeoff between correctness and hackability/readability... i think im just leaning for the latter :).  

.. but if you insist ill put the gaurdrails back on :)  let me know.

Reply:
I don't think I'm insisting anything here; I'm just trying to understand your reasoning for removing `set -e`.  I don't yet understand how it helps hackability or readability to do that; it may be obvious to you, but you need to use a few more words to explain it to me!

But anyway, I'm not going to hold up the PR on this point...